src/condor_scripts/annex/hpc.sh

```diff
@@ -447,15 +448,23 @@ if [[ -n $ALLOCATION ]]; then
     SBATCH_ALLOCATION_LINE="#SBATCH -A ${ALLOCATION}"
 fi
 
+if [[ $QUEUE_NAME =~ "q," ]]; then
+    QUEUE_NAME=${QUEUE_NAME/*q,/}
```

Comment:
What is this?

Reply:
A hack to avoid adding Yet Another Command-line Argument.  If `QUEUE_NAME` begins with `q,`, use the `-q` SLURM flag instead of `-p`.  It seemed better than either making a perlmutter-specific pilot script or having the pilot script be explicitly system-aware.

Reply:
Does this work?

Reply:
Yep. Note that the regex in the `if` statement checks for `q,` anywhere in the args but `${QUEUE_NAME/*q,/}` removes not only `q,` but everything before it. That won't be a problem in this case, but beware if you plan to use another hack like this in the future.

Reply:
Will do.  Thanks for the heads-up.